CONTRIBUTING.md

```diff
@@ -25,7 +25,7 @@ auto-update. If you modify files under `server/`, you'll have to re-run
 Note that when run this way, to aid in development, the server will
 auto-populate an event with a set of questions from a past live Q&A
 session I ran at
-<http://localhost:5173/#/event/00000000-0000-0000-0000-000000000000/secret>.
+<http://localhost:5173/event/00000000000000000000000000/secret>.
```

Comment:
There was a [move](https://github.com/jonhoo/wewerewondering/commit/275b649a2e34c90687a0333efebb55d095fcbf3f) from uuids to ulids.